File: Go/Fibonacci.go
package main

import "fmt"

func Fibonacci(n int) {
    n1 := 0
    n2 := 1
    n3 := 0
    for i := 1; i <= n; i++ {
        fmt.Println(n1)
        n3 = n1 + n2
        n1 = n2
        n2 = n3
    }
}

func main() {
    Fibonacci(10)
}

File: Java/Fibonacci.java
class Fibonacci {
    public static void main(String[] args) {
        int n = 10;
        Fibonacci(n);
    }
    static void Fibonacci(int n) {
        int n1 = 0;
        int n2 = 1;
        int n3 = 0;
        for (int i = 0; i < n; i++) {
            System.out.print(n1 + " ");
            n3 = n1 + n2;
            n1 = n2;
            n2 = n3;
        }
    }
}
